ros2launch/ros2launch/api/api.py

```diff
@@ -153,9 +153,12 @@ def launch_a_launch_file(
 
     # If 'launch-prefix' launch file argument is also provided in the user input,
     # the 'launch-prefix' option is applied since the last duplicate argument is used
-    if args is not None and args.launch_prefix is not None and len(args.launch_prefix) > 0:
+    if args.launch_prefix is not None and len(args.launch_prefix):
```

Comment:
@camm73 how about:
```suggestion
    if args and args.launch_prefix:
```
instead? `args` may still be `None` and using `len` instead of type coercion doesn't provide much of an advantage here IMHO. Same below.